[dbt tutorial] use vanilla jaffle shop in first section

sryza opened the pull request:
## Summary & Motivation

This PR is not independently mergeable, because downstream sections in the tutorial need to be updated to be consistent here.

Prior to this PR, the dbt tutorial asks the user to download a tutorial-specific scaffold that includes both a dbt project and an accompany Dagster project.  This PR updates the first section of the dbt tutorial to instead start with the vanilla dbt jaffle shop project, and doesn't scaffold any Dagster code.  The idea of this change is that separating the dbt setup steps from the Dagster setup steps will make it easier for users to learn how to set up Dagster with an existing dbt project.  I.e. they can essentially skip this section if they already have a working dbt project.

The followup tutorial work depends on @rexledesma adding `dagster-dbt project scaffold`, which the second section of the tutorial can then take advantage of.

## How I Tested These Changes

erinkcochran87 commented on docs/content/integrations/dbt/using-dbt-with-dagster/set-up-dbt-project.mdx:
```diff
@@ -11,62 +11,52 @@ description: Dagster can orchestrate dbt alongside other technologies.
 ---
 
-## Step 2: Create a profile
+## Step 2: Configure it to run with DuckDB
```
What's `it` here? dbt? Can you please make this heading more specific?

erinkcochran87 reviewed (commented):
Left comments for two small things, but otherwise looking good!

sryza commented on docs/content/integrations/dbt/using-dbt-with-dagster/set-up-dbt-project.mdx:
````diff
@@ -11,62 +11,52 @@ description: Dagster can orchestrate dbt alongside other technologies.
    ```
 
-3. Run the following to install the project's dependencies:
+3. Finally, download the sample dbt project into that folder. This step requires git to be installed on your computer. If it's not already installed, you can install it using the [intructions on the git website](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git).
````
That makes sense – I took this out and updated the pre-requisites instead.

sryza commented:
@erinkcochran87 mind taking another look at this one? I incorporated your suggestions and added one new change, which is to take out "download the [`tutorial_dbt_dagster`](https://github.com/dagster-io/dagster/tree/master/examples/tutorial_dbt_dagster) Dagster example." prerequisite, because the changes here make that example no longer used.

erinkcochran87 reviewed (approved):
Looks great Sandy!
